CondCore/CTPPSPlugins/test/testPPSTimingCalibration.cpp

```diff
@@ -30,11 +30,7 @@ int main(int argc, char** argv) {
 
   edm::LogPrint("testPPSCalibrationPI") << "## Exercising TimingCalibration plots ";
 
-  ParametersPerChannel<PPSTimingCalibrationPI::db0,
-                       PPSTimingCalibrationPI::plane0,
-                       PPSTimingCalibrationPI::parameter0,
-                       PPSTimingCalibration>
-      test;
+  ParametersPerChannel<PPSTimingCalibrationPI::parameter0, PPSTimingCalibration> test;
```

Comment:
now that the class `ParametersPerChannel` accepts input parameters, I think the test should be equipped to comply with that.

e.g. this change

```diff
diff --git a/CondCore/CTPPSPlugins/test/testPPSTimingCalibration.cpp b/CondCore/CTPPSPlugins/test/testPPSTimingCalibration.cpp
index 262a9779e58..76d75dc7034 100644
--- a/CondCore/CTPPSPlugins/test/testPPSTimingCalibration.cpp
+++ b/CondCore/CTPPSPlugins/test/testPPSTimingCalibration.cpp
@@ -31,7 +31,19 @@ int main(int argc, char** argv) {
   edm::LogPrint("testPPSCalibrationPI") << "## Exercising TimingCalibration plots ";
 
   ParametersPerChannel<PPSTimingCalibrationPI::parameter0, PPSTimingCalibration> test;
+  py::dict inputs;
+  inputs["db (0,1)"] = "0";
+  inputs["station (1,2)"] = "1";
+  inputs["plane (0-3)"] = "0";
+  test.setInputParamValues(inputs);
+
   test.process(connectionString, PI::mk_input(tag, start, end));
   edm::LogPrint("testparametersPerChannel") << test.data();
+
+  inputs.clear();
+#if PY_MAJOR_VERSION >= 3
+  Py_INCREF(inputs.ptr());
+#endif
+
   Py_Finalize();
 }
```

makes the test pass.

Reply:
@mmusich thanks for your suggestions -  I applied this fix and committed the changes.